controllers/v1/users/user.controller.js

```diff
@@ -77,6 +77,36 @@ const getUserAttendance = async (request, response) => {
   }
   return response.json(responseData)
 }
+const getUserEventsAttendance = async (request, response) => {
+  const userService = await serviceContainer('users')
+  const eventService = await serviceContainer('events')
+
+  const userId = request.user.id
+  let responseData
+
+  if (!userId) {
+    return response.status(400).json(baseController.getErrorResponse('Parameters are missing'))
+  }
+
+  try {
+    const { status, data, message } = await userService.fetchUserEventsAttendance(
+      userId,
+      eventService
+    )
+
+    responseData = {
+      status,
+      data,
+      message,
+    }
+
+  } catch (err) {
+    return response
+      .status(500)
+      .json(baseController.getErrorResponse('Error verifying user attendance'))
+  }
+  return response.status(200).json(responseData)
```

Comment:
What about doing this small refactor

```suggestion
  if (!userId) {
    return response.status(400).json(baseController.getErrorResponse('Parameters are missing'))
  }

  try {
    const { status, data, message } = await userService.fetchUserEventsAttendance(
      userId,
      eventService
    )

    return {
      status,
      data,
      message,
    }

  } catch (err) {
    return response
      .status(500)
      .json(baseController.getErrorResponse('Error verifying user attendance'))
  }
```

Reply:
Applied! 👍🏼